toqito/channels/partial_trace.py

```diff
@@ -189,10 +189,10 @@ def partial_trace(
         sys = np.array(sys)
     if isinstance(sys, int):
         sys = np.array([sys])
-    set_diff = list(set(list(range(1, num_sys + 1))) - set(sys + 1))
 
+    set_diff = list(set(list(range(0, num_sys))) - set(sys))
```

Comment:
```suggestion
    set_diff = list(set(list(range(num_sys))) - set(sys))
```
Considering `list(range(4))` is equivalent to `list(range(0, 4))`, you don't need to specify the starting index every time if you want to start from 0.

Reply:
Agreed